configure.sh

```diff
@@ -12,50 +12,62 @@ set -e -u
 
 BASE_PATH=$(cd "$(dirname "$0")"; pwd)
 
-PRIVATE_HEADER="$BASE_PATH/private.h"
 DEFAULT_PRIVATE_HEADER="$BASE_PATH/private_default.h"
+PRIVATE_HEADER="private.h"
+PRIVATE_PROPERTIES="android/secure.properties"
+PRIVATE_NETWORK_CONFIG="android/res/xml/network_security_config.xml"
+PRIVATE_GOOGLE_SERVICES="android/google-services.json"
 
-PRIVATE_PROPERTIES="$BASE_PATH/android/secure.properties"
-PRIVATE_NETWORK_CONFIG="$BASE_PATH/android/res/xml/network_security_config.xml"
-PRIVATE_GOOGLE_SERVICES="$BASE_PATH/android/google-services.json"
 SAVED_PRIVATE_REPO_FILE="$BASE_PATH/.private_repository_url"
```

Comment:
Why BASE_PATH was removed above but not here?

Reply:
its because e.g. PRIVATE_HEADER is printed out in usage() and having a short path there is cleaner-looking and more readable to the user